src/css/profile/mobile/common/scrollview.less

```diff
@@ -94,22 +94,24 @@
 
 .ui-overflow-indicator-top,
 .ui-overflow-indicator-bottom {
-	position: absolute;
+	position: fixed;
 	display: none;
-	width: 100%;
-	height: 29 * @unit_base;
+	width: 10%;
+    transform: scaleX(10);
+    transform-origin: left;
```

Comment:
Fix indent

Reply:
Done.

Thank you for review. I think I fixed all issues found by you.